src/DurableTask.Core/Common/Utils.cs

```diff
@@ -62,6 +62,90 @@ public static class Utils
             Binder = new PackageUpgradeSerializationBinder()
 #endif
         };
+        private static readonly JsonSerializer objectJsonSerializer = JsonSerializer.Create(ObjectJsonSettings);
```

Comment:
```suggestion
        private static readonly JsonSerializer DefaultObjectJsonSerializer = JsonSerializer.Create(ObjectJsonSettings);
```

And please move this up with the other `static readonly` fields.

Reply:
I have the same question as in another comment, which I'll paste here:

I applied the renaming, but I was unsure about your suggestion on moving this further up, as it seems to me that all the fields preceding DefaultSerializer are already static readonly, and near the top of the class. Am I missing something?

Reply:
I think I misread the code, thinking that the line above this was a method when really it was an object initializer for other field (I should make more use of the github.dev PR experience so I can better see the full context of these changes).